dsb-gui/src/test/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/FileTreeTableItemTest.java

```diff
@@ -83,23 +68,19 @@ public void refresh() throws Exception {
         new JFXPanel();
         Platform.runLater(() -> {
             try {
-                final ClassLoader classLoader = Ds3GetJob_Test.class.getClassLoader();
-                final URL url = classLoader.getResource(SessionConstants.LOCAL_FOLDER);
-                Path path = null;
-                if (url != null) {
-                    path = new File(url.getFile()).toPath();
-                }
+                final Path path = ResourceUtils.loadFileResource(SessionConstants.LOCAL_FOLDER);
+                final File testFolder = path.toFile();
+                final int numFiles = testFolder.list().length;
```

Comment:
We should look to see if there is an alternate nio implementation to do the same thing.